packages/api-explorer/src/components/SchemaTabs/index.jsx

```diff
@@ -92,14 +93,23 @@ export default class SchemaTabs extends Component {
   }
 
   renderSchemaExample() {
-    const {schema} = this.state
-    return (
-      <JsonViewer
-        missingMessage={'schemaTabs.missing.example'}
-        schema={schema.example}
-        key={'json-viewer-example'}
-      />
-    )
+    try {
+      const {schema} = this.state
+      let example = get(schema, EXAMPLE)
+      if (!example) {
+        example = jsf.generate(schema)
+      }
+
+      return (
+        <JsonViewer
+          missingMessage={'schemaTabs.missing.example'}
+          schema={example}
+          key={'json-viewer-example'}
+        />
+      )
+    } catch (error) {
+      return <Alert type={'error'} message={error.message} />
+    }
```

Comment:
evitiamo il ritorno nel try/catch: l'istanziazione di un compontent react potrebbe lanciare un'eccezione e renderizzare cose a caso.

Reply:
I’m not sure if the problem is the use of React components within try/catch block or the not friendly error shown to the user

Reply:
The alert is ok. Not put it in the catch.
```js
let ret = <Alert type={'error'} message={error.message} />
try {
  ...
} catch (e) {
  ... // maybe a log here?
}

return ret
```

Reply:
I not have an `error.message` outside of catch or we prefer to use a formattedMessage error for this case to inform the users? Inside the catch would you use a `console.log` of `e`?

Reply:
Why not using an ErrorBoundary component instead?

Reply:
but what is this catch for? What is the function that can throw here? Because previously the `JsonViewer` was not inside a try/catch statement.

Reply:
the "try / catch" must ensure that jsf.generate does not explode in the event of a malformed "schema"

Reply:
> Why not using an ErrorBoundary component instead?

Because we want to show the rest of the endpoint view without blocking the ability to use the component form